sql/core/src/main/scala/org/apache/spark/sql/execution/python/ExtractPythonUDFs.scala

```diff
@@ -132,7 +132,7 @@ private[spark] object ExtractPythonUDFs extends Rule[SparkPlan] {
         val validUdfs = udfs.filter { case udf =>
           // Check to make sure that the UDF can be evaluated with only the input of this child.
           udf.references.subsetOf(child.outputSet)
-        }
+        }.toArray  // force the iterator to an array, that can't be serialized in Scala 2.10
```

Comment:
do u mean the iterator couldn't have been serialized in scala 2.10?

maybe say "Turn it into an array since iterators cannot be serialized in Scala 2.10"